viewmodel-events/common/src/main/kotlin/Event.kt

```diff
@@ -0,0 +1,4 @@
+package com.redmadrobot.gears.viewmodelevents
```

Comment:
`gears` could be dropped from the package
```suggestion
package com.redmadrobot.viewmodelevents
```